packages/core/src/state-management/repository.ts

```diff
@@ -77,9 +124,25 @@ export class Repository {
     await this.#map.delete(docId.toString());
   }
 
+  async list(docId?: DocID): Promise<string[]> {
```

Comment:
Can we call this `listPinned` to be clearer?

Reply:
k